# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/fetch/FetchTypes.ipdlh
# Commit: d77b78a71a7e
# Full Hash: d77b78a71a7e76f975052eee5c2fef7baeb88d44
# Author: Eden Chuang <echuang@mozilla.com>
# Date: 2023-01-20 21:21:03
# Regressor Bug: 1351231
# File Overlap Count: 1
# Description:
#   Bug 1351231 - Support conversation between InternalRequest and IPCInternalRequest. r=dom-worker-reviewers,jesup
#   
#   Depends on D138812
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D138813
# ==============================================================================

diff -r dcfa6d7e82c0 -r d77b78a71a7e dom/fetch/FetchTypes.ipdlh
--- a/dom/fetch/FetchTypes.ipdlh	Fri Jan 20 09:09:16 2023 +0000
+++ b/dom/fetch/FetchTypes.ipdlh	Fri Jan 20 09:09:17 2023 +0000
@@ -18,6 +18,7 @@
 using ResponseType from "mozilla/dom/ResponseBinding.h";
 using struct nsID from "nsID.h";
 using FetchDriverObserver::EndReason from "mozilla/dom/FetchDriver.h";
+using nsILoadInfo::CrossOriginEmbedderPolicy from "nsILoadInfo.h";
 [RefCounted] using class mozilla::RemoteLazyInputStream from "mozilla/RemoteLazyInputStream.h";
 [RefCounted] using class nsITransportSecurityInfo from "nsITransportSecurityInfo.h";
 
@@ -59,6 +60,7 @@
   uint32_t contentPolicyType;
   nsString referrer;
   ReferrerPolicy referrerPolicy;
+  ReferrerPolicy environmentReferrerPolicy;
   RequestMode requestMode;
   RequestCredentials requestCredentials;
   RequestCache cacheMode;
@@ -70,6 +72,7 @@
   uint32_t interceptionContentPolicyType;
   RedirectHistoryEntryInfo[] interceptionRedirectChain;
   bool interceptionFromThirdParty;
+  CrossOriginEmbedderPolicy embedderPolicy;
 };
 
 struct InternalResponseMetadata {